src/main/java/it/eng/idsa/businesslogic/processor/common/GetTokenFromDapsProcessor.java

```diff
@@ -25,9 +25,9 @@
  */
 
 @Component
-public class SenderGetTokenFromDapsProcessor implements Processor {
+public class GetTokenFromDapsProcessor implements Processor {
```

Comment:
Can you fix this 
if (eccHttpSendRouter.equals("http-header"))
as we discussed before? :)

Same change is needed for ValidateProcessor

Reply:
I fixed that part in both processor